### PR TITLE
Add scm type to telemetry

### PR DESCRIPTION
**Pull request by nturinski**

### CODE REVIEW

File: appservice/src/SiteWrapper.ts [hunk 1]
```diff
@@ -183,12 +183,13 @@ export class SiteWrapper {
     }
 
     public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, configurationSectionName: string, confirmDeployment: boolean = true, telemetryProperties?: TelemetryProperties): Promise<void> {
+        const config: SiteConfigResource = await this.getSiteConfig(client);
         if (telemetryProperties) {
             telemetryProperties.sourceHash = randomUtils.getPseudononymousStringHash(fsPath);
             telemetryProperties.destHash = randomUtils.getPseudononymousStringHash(this.appName);
+            telemetryProperties.scmType = config.scmType;
```
Comment (StephenWeatherford) [inline appservice/src/SiteWrapper.ts]: Anything else we want to add that's easily available?

Comment (nturinski) [reply appservice/src/SiteWrapper.ts]: I just pushed up slots

Comment (EricJizbaMSFT) [reply appservice/src/SiteWrapper.ts]: The confirmation for zip deployments is in this file - so if you want to track where people drop off - maybe add a boolean there?

Comment (StephenWeatherford) [reply appservice/src/SiteWrapper.ts]: Add it as a cancelStep property, e.g.:

telemetryProperties.cancelStep = 'confirmDeployment';

**Pull request merged**